spanner/errors.go

```diff
@@ -26,6 +27,11 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+var (
+	// ErrRecordNotFound record not found error
+	ErrRecordNotFound = errors.New("record not found")
```

Comment:
`ErrRowNotFound` corresponds better with the original error message `row not found...`

```suggestion
	ErrRowNotFound = errors.New("record not found")
```